detekt.yml

```diff
@@ -0,0 +1,425 @@
+build:
+  maxIssues: 2
+  weights:
+    complexity: 3
+
+processors:
+  active: true
+
+console-reports:
+  active: true
+
+comments:
+  active: true
+  excludes: "**/test/**,**/*.Test.kt,**/*.Spec.kt,**/*.Spek.kt"
+  CommentOverPrivateFunction:
+    active: false
+  CommentOverPrivateProperty:
+    active: false
+  EndOfSentenceFormat:
+    active: false
+    endOfSentenceFormat: ([.?!][ \t\n\r\f<])|([.?!:]$)
+  UndocumentedPublicClass:
+    active: false
+    searchInNestedClass: true
+    searchInInnerClass: true
+    searchInInnerObject: true
+    searchInInnerInterface: true
+  UndocumentedPublicFunction:
+    active: false
+  UndocumentedPublicProperty:
+    active: false
+
+complexity:
+  active: true
+  ComplexCondition:
+    active: true
+    threshold: 4
+  ComplexInterface:
+    active: false
+    threshold: 10
+    includeStaticDeclarations: false
+#  LabeledExpression:
+#    active: true
+#    ignoredLabels: []
+  LargeClass:
+    active: true
+    threshold: 500
+    excludes: "**/test/**,**/*.Test.kt,**/*.Spec.kt,**/*.Spek.kt"
+  LongMethod:
+    active: true
+    threshold: 60
+    excludes: "**/test/**,**/*.Test.kt,**/*.Spec.kt,**/*.Spek.kt"
+#  LongParameterList:
```

Comment:
I suggest either removing these comment blocks or uncommenting them

Reply:
Okey, i'll remove it